### PR TITLE
split out NTS again

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: pipelines/release/weekly_release.groovy [hunk 1]
```diff
@@ -176,13 +176,25 @@ notify.wrap {
       //  stages within other parallel stages, so we do these all
       //  sequentially in order not to have them held up by validate_drp.
 
-      stage('SAL-sciplat-lab linked image set (master/summit/base/kueyen/nts/tts') {
+      stage('SAL-sciplat-lab linked image set (master/summit/base/kueyen/tts') {
```
Comment (ktlim) [inline pipelines/release/weekly_release.groovy:179]: Didn't notice the unclosed paren before, but might be nice to fix it now.

File: pipelines/release/weekly_release.groovy [hunk 2]
```diff
@@ -176,13 +176,25 @@ notify.wrap {
...
             parameters: [
               string(name: 'TAG', value: eupsTag),
-              string(name: 'ENVIRONMENTS', value: 'master summit base kueyen nts tts'),
+              string(name: 'ENVIRONMENTS', value: 'master summit base kueyen tts'),
+            ],
+            wait: false,
+          )
+        } // retry
+      } // stage
+      stage('SAL-sciplat-lab linked image set (nts') {
```
Comment (ktlim) [inline pipelines/release/weekly_release.groovy:191]: Same close-paren needed here.

**Review by ktlim — APPROVED** [review]

File: pipelines/release/weekly_release.groovy (hunk 1 above)
Comment (athornton) [reply pipelines/release/weekly_release.groovy:179]: TY

File: pipelines/release/weekly_release.groovy (hunk 2 above)
Comment (athornton) [reply pipelines/release/weekly_release.groovy]: Done

File: pipelines/release/weekly_release.groovy (hunk 1 above)
Comment (athornton) [reply pipelines/release/weekly_release.groovy]: Done

**Pull request merged**